.gitignore

```diff
@@ -28,6 +28,7 @@
 .kconfig
 *.lock
 /bin
+/bin_debug
```

Comment:
should we name it symbols?

Reply:
Initially I wrote about two possibilities but then deleted because `bin_debug` is quite self-explanatory :-P

1. `bin/release`, `bin/devel`, `bin/debug` so all is under `bin/` and then subdiretory (more elegant).
2. `bin/release-$TIMESTAMP`, `bin/debug-$TIMESTAMP` so all is just under `bin/` (less elegant).

Reply:
OK, let's keep it.